pages/nominations.jsx

```diff
@@ -43,6 +45,15 @@ const Nominations = () => {
     setShowModal(!showModal);
   };
 
+  if (!showNominations) {
+    return (
+      <div className="container text-center">
+        <h1>Nominations are currently closed.</h1>
+        <p>Come back next year for the nominations!</p>
+      </div>
+    );
+  }
+
   return (
     <>
```

Comment:
Currently lacking a title, would be nice to fix in this PR

```suggestion
    <>
      <Head>
        <title>Nominations | UNSW Co-op Society</title>
      </Head>
```